app/react-native/src/preview/components/StoryListView/index.js

```diff
@@ -88,24 +116,33 @@ export default class StoryListView extends Component {
     const { data } = this.state;
 
     return (
-      <SectionList
-        testID="Storybook.ListView"
-        style={[style.list]}
-        renderItem={({ item }) => (
-          <ListItem
-            title={item.name}
-            kind={item.kind}
-            selected={item.kind === selectedKind && item.name === selectedStory}
-            onPress={() => this.changeStory(item.kind, item.name)}
-          />
-        )}
-        renderSectionHeader={({ section: { title } }) => (
-          <SectionHeader title={title} selected={title === selectedKind} />
-        )}
-        keyExtractor={(item, index) => item + index}
-        sections={data}
-        stickySectionHeadersEnabled={false}
-      />
+      <React.Fragment>
```

Comment:
Maybe lets not use fragment yet? They are only added in React 16.2. It will break backwards compatibility.